Makefile

```diff
@@ -62,7 +62,7 @@ checkindent: ## Ensure that indentation is correct
 	    emacs -Q --batch \
 	        --eval "(setq inhibit-message t)" \
 	        --eval "(load (expand-file-name \"el-patch.el\") nil t)" \
```

Comment:
Should I load `el-patch-template.el` here too? I am not sure why `el-patch.el` itself is loaded.

Reply:
The reason for `el-patch.el` being loaded is to get the macros defined. This is because macros are indented differently than regular functions, and if the macro definition isn't loaded, then Emacs does not know how to indent it correctly. You only need to load `el-patch-template.el` if that file defines a macro which has indentation rules different than a regular function.